### DISCUSSION TITLE
Fitting DataFrames within book margins

### DISCUSSION THREAD

**Discussion by edgarsmdn** [executablebooks/meta#957] [Q&A] [answered]
Hi there!

Does anyone know a way to keep DataFrames within the book margins?
For example, look at the DataFrame here: https://inferentialthinking.com/chapters/06/3/Example_Population_Trends.html#example-population-trends

It is very wide, and gets over the margins making it not very nice-looking

Thank you in advance!

**Comment by choldgraf** [comment]
This should be fixed once we update jupyter book to use the 1.0 version of the book theme 👍

https://sphinx-book-theme.readthedocs.io has the latest release, it just hasn't been bumped in jupyter book yet.

Cc @mmcky in case he has cycles to make another release 🙂

**Reply by edgarsmdn** [reply]
Thanks for the answer! Looking forward for the update :)

**Reply by agoose77** [reply]
There's now been a `jupyter-book==0.15.0` release which uses `sphinx-book-theme==1.0.0`. Can you report back to see whether it's fixed your issue? :)

**Reply by edgarsmdn** [reply]
Thank you! That fixed it! 😃

**Comment by edgarsmdn** [comment, marked as answer]
Awesome!! With `jupyter-book==0.15.0` the DataFrames are now displayed within the margins :)
Also, I really like the new layout and the light/dark mode
Thanks!
